common/src/thrift/file_system_master.thrift

```diff
@@ -36,12 +38,22 @@ struct CreateTOptions {
   4: optional i64 ttl
 }
 
+struct PersistCommand {
```

Comment:
move this definition below MkdirTOptions so that it is next to PersistFile definition (and we keep definition alphabetically sorted)